README.markdown

```diff
@@ -216,6 +216,19 @@ nvm install 6 --reinstall-packages-from=5
 nvm install v4.2 --reinstall-packages-from=iojs
 ```
 
+### Default global packages from file while installing
+
+If you have a list of default packages you want installed every time you install a new version we support that too. You can add any valid package, version, repo, url for installing an npm package.
```

Comment:
"for installing an npm package - anything npm would accept as a package argument on the command line."?